src/utils/theme.ts

```diff
@@ -0,0 +1,16 @@
+import { STORAGE_KEYS } from "./constant";
+
+const mediaPrefersDarkScheme = window.matchMedia(
+  "(prefers-color-scheme: dark)"
+);
+mediaPrefersDarkScheme.addEventListener("change", () => updateTheme());
+
+export const updateTheme = () => {
+  const currentThemeType = localStorage.getItem(STORAGE_KEYS.THEME_TYPE);
```

Comment:
It seems we should add make `"system"` as fallback value here. Maybe we can use an Enum for theme type values now.

Reply:
I don't think create ENUM is worth it by a second look, since line 16 and  17 should actually be string value.

I will make `system` as default for `currentThemeType` if that's what's you are asking